build/CreateInsertionMetadata.proj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.Build.NoTargets/1.0.94" DefaultTargets="GenerateInsertionParameters">
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="GenerateInsertionParameters">
```

Comment:
Why make this change?  This project didn't use any of the standard build tasks, so the NoTargets SDK was chosen specifically.

Reply:
"In the newest builds" doesn't make sense.  It's a [NuGet package](https://www.nuget.org/packages/Microsoft.Build.NoTargets), not part of the .NET SDK.  Can you share the error you were seeing?  I tried building it from your branch but with this change reverted and it seemed to build fine.

Reply:
After running commands like CI would, the existing SDK reference works great.